### PR TITLE
Change project to Django ORM only

### PR DESCRIPTION
**Pull request by jfrverdasca**
## What this PR change:

* Now project uses only Django ORM
* SQLAlchemy related code removed
* SQLAlchemy and it's dependencies removed
* Changed all SQLAlchemy related queries to Django queries ensuring the same results
* Test fix

## Reviewers:

- [ ] @kenvontucky

## Issues:

* #144 

## Print screens showing the changes:

NA

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -20,12 +20,12 @@ gitpython = "^3.1.43"
 python-json-logger = "^2.0.7"
 pathspec = "^0.12.1"
 pgvector = "^0.3.2"
+pytest-postgresql = "^6.0.0"
```
Comment (kenvontucky) [inline pyproject.toml:23]: this should be in dev dependencies

**Review by kenvontucky — CHANGES_REQUESTED** [review]

**Review by kenvontucky — APPROVED** [review]

File: labs/api/codemonkey_endpoints.py [hunk 2]
```diff
@@ -94,41 +95,47 @@ async def create_branch_endpoint(request: HttpRequest, create_branch: CreateBran
...
+    return await sync_to_async(prepare_prompt_and_context_task, thread_sensitive=True)(
         issue_body=prepare_prompt_and_context.issue_body, embeddings=prepare_prompt_and_context.embeddings
     )
 
 
 @router.post("/get_llm_response")
 @async_time_and_log_function
 async def get_llm_response_endpoint(request: HttpRequest, get_llm_reponse: GetLLMResponseRequest):
-    return get_llm_response_task(prepared_context=get_llm_reponse.context)
+    return await sync_to_async(get_llm_response_task, thread_sensitive=True)(prepared_context=get_llm_reponse.context)
```
Comment (JdFSilva) [inline labs/api/codemonkey_endpoints.py:124]: why do we need `sync_to_async` and `thread_sensitive` ?

File: .github/workflows/ci_cd.yml [hunk 3]
```diff
@@ -43,10 +43,6 @@ jobs:
       - name: Poetry Install
         uses: knowsuchagency/poetry-install@v2
 
-      - name: Create Tables
```
Comment (JdFSilva) [inline .github/workflows/ci_cd.yml:46]: should also remove `TEST_ENVIRONMENT` from the `env` at the start of this file

File: labs/embeddings/base.py [hunk 4]
```diff
@@ -23,42 +22,31 @@ def __init__(self, embedder: type["Embedder"], *args, **kwargs):
     def embed(self, prompt, *args, **kwargs) -> Embeddings:
```
Comment (JdFSilva) [inline labs/embeddings/base.py:22]: you should call this file `embedder.py` since that's the only thing that this file has

**Review by JdFSilva — APPROVED** [review]

**Pull request merged**